serverless.yml

```diff
@@ -1,228 +1,237 @@
+      Resource: '*'
+#resources:
+#  Resources:
+#    UserPool:
+#      Type: AWS::Cognito::UserPool
+#      Properties:
+#        UserPoolName: ${file(env.yml):${opt:stage, self:provider.stage}.USER_POOL}
+#        UsernameAttributes: [email]
+#        AutoVerifiedAttributes:
+#          - email
+#        Schema:
+#          - Name: email
+#            AttributeDataType: String
+#            Mutable: false
+#            Required: true
+#        Policies:
+#          PasswordPolicy:
+#            RequireLowercase: true
+#            RequireSymbols: false
+#            RequireNumbers: true
+#            MinimumLength: 8
+#            RequireUppercase: true
+#        AdminCreateUserConfig:
+#          UnusedAccountValidityDays: 60
+#    UserPoolClient:
+#      Type: AWS::Cognito::UserPoolClient
+#      Properties:
+#        ClientName: ${self:provider.environment.USER_POOL_CLIENT}
+#        GenerateSecret: false
+#        UserPoolId:
+#          Ref: UserPool
+#  Outputs:
+#    UserPoolId:
+#      Value:
+#        Ref: UserPool
+#      Export:
+#        Name: ${opt:stage, self:provider.stage}UserPool::Id
+#    UserPoolClientId:
+#      Value:
+#        Ref: UserPoolClient
+#      Export:
+#        Name: ${opt:stage, self:provider.stage}UserPoolClient::Id
+
+package:
+  include:
+    - bin/*
+custom:
+  queuesUrls:
+    SubscribeQueue:
+      local: ''
+      dev: !Ref SubscribeQueue
+      test: !Ref SubscribeQueue
+      prod: !Ref SubscribeQueue
+  queuesNames:
+    SubscribeQueue:
+      local: 'win-local-subscribe'
+      dev: 'win-dev-subscribe'
+      test: 'win-test-subscribe'
+      prod: 'win-prod-subscribe'
+  tablesNames:
+    IconikTokensTable:
+      local: 'IconikTokens-local'
+      dev: 'IconikTokens-dev'
+      test: 'IconikTokens-test'
+      prod: 'IconikTokens'
+    UsersTable:
+      local: 'Users-local'
+      dev: 'Users-dev'
+      test: 'Users-test'
+      prod: 'Users'
+    JobsTable:
+      local: 'Jobs-local'
+      dev: 'Jobs-dev'
+      test: 'Jobs-test'
+      prod: 'Jobs'
+  customActionsURLs:
+    GetMediaInfo:
+      local: '${file(env.yml):local.OFFLINE_API_BASE_URL}api/media-info'
+      dev: !Sub
+        - 'https://${gatewayId}.execute-api.${region}.${suffix}/${opt:stage, self:provider.stage}/${path}'
+        - gatewayId: !Ref ApiGatewayRestApi
+          region: !Ref AWS::Region
+          suffix: !Ref AWS::URLSuffix
+          path: 'api/media-info'
+      test: !Sub
+        - 'https://${gatewayId}.execute-api.${region}.${suffix}/${opt:stage, self:provider.stage}/${path}'
+        - gatewayId: !Ref ApiGatewayRestApi
+          region: !Ref AWS::Region
+          suffix: !Ref AWS::URLSuffix
+          path: 'api/media-info'
+      prod: !Sub
+        - 'https://${gatewayId}.execute-api.${region}.${suffix}/${opt:stage, self:provider.stage}/${path}'
+        - gatewayId: !Ref ApiGatewayRestApi
+          region: !Ref AWS::Region
+          suffix: !Ref AWS::URLSuffix
+          path: 'api/media-info'

```

Comment:
You can replace GetMediaInfo URLs with your example CA because GetMediaInfo isn't a CA and located here just as an example.